sling/core/pckgmgr/src/main/java/com/composum/sling/core/pckgmgr/PackageServlet.java

```diff
@@ -211,6 +216,22 @@ public ResourceHandle getResource(SlingHttpServletRequest request) {
     // operation implementations
     //
 
+    protected class ListOperation implements ServletOperation {
```

Comment:
flat structure is needed, because external tooling I made requires to obtain somehow complete list of installed packages. Tree operation requires to have specified package group to have rendered package definition. That constraint could be alternatively relaxed, however to be more sure about backward compatibility new endpoint is introduced. Also tree structure is just not so trivial to parse. That  new endpoint simplifies that.